Logging.podspec

```diff
@@ -0,0 +1,41 @@
+def self.infer_version_from_git
+  return nil unless Dir.exist?('.git')
+
+  `git tag --list '*.*.*' --sort=committerdate`.split.select do |tag|
```

Comment:
Couldn’t we sort this ‘by version’? Just in case we tag 1.2.0 and later we need to add a 1.1.2. I think `sort` can sort by version and git might be able to too?

Reply:
Thanks for pointing it out! 

I've replaced it with a git command that correctly looks for the tags in the current commit tree only.

Reply:
This way if you have 1.2.0 and you later branch 1.1.2, if will get handled correctly.